pom.xml

```diff
@@ -102,6 +87,68 @@
 				<groupId>org.springframework.boot</groupId>
 				<artifactId>spring-boot-maven-plugin</artifactId>
 			</plugin>
+			<plugin>
```

Comment:
why do you need all these plugins? Spring Boot already builds a fat jar

Reply:
Really? I saw in 'target' just separated compiled classes without this plugins

Reply:
My mistake, I understood